UnitTests/Utilities/GeometryModelRenderer/src/GeometryModelRenderer.h

```diff
@@ -22,7 +22,6 @@
 #include <buw.Engine.h>
 
 #include "OffConverter/OffModel.h"
```

Comment:
This still seems like an overkill. Is `OffModel` really needed?

Reply:
I tried to remove it and also tried moving it to the OffGeomeryModelRenderer and other files but all options I tried throw errors in the GeometryModelRenderer.h file.
So it looks like it really is needed here.

Reply:
That seems weird, but thanks for double checking!